aspnetcore/release-notes/aspnetcore-10/includes/jsonPatch.md

```diff
@@ -0,0 +1,240 @@
+### New JsonPatch Implementation with System.Text.Json
+
+**[JSON Patch](https://jsonpatch.com/)**:
+
+* Is a standard format for describing changes to apply to a JSON document.
+* Is defined in [RFC 6902] and is widely used in RESTful APIs to perform partial updates to JSON resources.
+* Represents a sequence of operations (for example, Add, Remove, Replace, Move, Copy, Test) that can be applied to modify a JSON document.
+
+In web apps, JSON Patch is commonly used in a PATCH operation to perform partial updates of a resource. Rather than sending the entire resource for an update, clients can send a JSON Patch document containing only the changes. Patching reduces payload size and improves efficiency.
+
+[RFC 6902]: https://tools.ietf.org/html/rfc6902
+
+This release introduces a new implementation of [`JsonPatch`](/dotnet/api/system.text.json based on `System.Text.Json` serialization. This feature:
```

Comment:
I think there's something wonky on this line ... it looks like the link is not closed. Maybe it should be
```suggestion
This release introduces a new implementation of `JsonPatch` based on `System.Text.Json` serialization. This feature:
```